src/password-hash.lisp

```diff
@@ -4,8 +4,7 @@
 (defun make-random-salt (&optional (size 16))
   "Generate a byte vector of SIZE (default 16) random bytes, suitable
 for use as a password salt."
-  (let ((prng (or *prng* (make-prng :fortuna :seed :random))))
-    (random-data size prng)))
+  (random-data size))
```

Comment:
With the default OS PRNG, there's no need to instantiate a random one each time.

This also has the advantage of not blocking on Linux, which has a broken `/dev/random`